xcm/pallet-xcm/src/lib.rs

```diff
@@ -550,6 +550,52 @@ pub mod pallet {
 			)
 		}
 
+		/// Transfer some assets from sovereign account to reserve holder chain and
+		/// forward a notification XCM.
```

Comment:
I don't believe this extrinsic actually forwards a notification XCM. Were you talking about the `ReserveAssetDeposited` instruction? All that `dest` receives is this:

```
[
    WithdrawAssets(assets),
    ClearOrigin,
    BuyExecution { fees, weight_limit },
    DepositAsset { assets: All, max_assets, beneficiary }
]
```

The `ReserveAssetDeposited` instruction is nowhere to be found here.

Reply:
I used common term, sorry for misleading. The XCM you paste is the notification.

Reply:
But `ReserveAssetDeposited` is not emitted by your code; this documentation is then wrong.

Reply:
No problem, let me change it.